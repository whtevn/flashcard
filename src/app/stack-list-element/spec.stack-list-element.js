import renderer from 'react-test-renderer';
import React from 'react';
import MuiThemeProvider from 'material-ui/styles/MuiThemeProvider';
import FCD_StackListElement from './component.stack-list-element.js';
import { List } from 'immutable';

test('stack list element renders correctly', () => {
  const tree = renderer.create(
      <MuiThemeProvider>
        <FCD_StackListElement name="element name" onSelect={()=>{}} cards={List()} id={0} />
      </MuiThemeProvider>
      ).toJSON();
  expect(tree).toMatchSnapshot();
});

