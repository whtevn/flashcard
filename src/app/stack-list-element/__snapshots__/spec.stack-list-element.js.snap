// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`stack list element renders correctly 1`] = `
<div>
  <a
    href="#"
    onClick={[Function]}
  >
     
    element name
     
  </a>
</div>
`;
